build/Dockerfile

```diff
@@ -1,19 +1,8 @@
-# We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
-# to install it.
-FROM registry.access.redhat.com/ubi8/ubi-minimal as build
+FROM docker.io/openshift/origin-release:golang-1.15 as build
 LABEL stage=build
-RUN microdnf -y install rsync make git tar findutils diffutils && microdnf clean all
 RUN mkdir /build/
 
-# Install go 1.14
-RUN mkdir /build/go
-WORKDIR /build/go
-RUN curl https://golang.org/dl/go1.14.7.linux-amd64.tar.gz -L -o go.tar.gz
-# Check sha256
-RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.tar.gz" |sha256sum -c
-RUN tar -C /usr/local -xzf go.tar.gz
 ENV GOBIN=/usr/local/go/bin
```

Comment:
This is not needed AFAICT or did you find out otherwise?

Reply:
ah yea this won't be needed when using the builder image, I think.